package.json

```diff
@@ -1,50 +1,53 @@
 {
-  "name": "@most/mostPackage",
-  "version": "1.0.0",
-  "description": "mostPackage",
-  "main": "dist/mostPackage.js",
+  "name": "@most/package-starter",
+  "version": "0.0.0",
+  "description": "A well-crafted starting place for mostjs packages.",
+  "main": "dist/package-starter.js",
   "files": [
-    "dist/mostPackage.js"
+    "dist/package-starter.js"
   ],
   "scripts": {
-    "build-dist": "mkdir -p dist && rollup src/index.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/mostPackage' -o dist/mostPackage.js",
-    "build": "npm run build-dist && uglifyjs dist/mostPackage.js -o dist/mostPackage.min.js",
+    "build-dist": "mkdir -p dist && rollup src/index.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/package-starter' -o dist/package-starter.js",
+    "build": "npm run build-dist && uglifyjs dist/package-starter.js -o dist/package-starter.min.js",
     "prepublish": "npm run build",
     "preversion": "npm run build",
     "unit-test": "babel-node ./node_modules/.bin/isparta cover _mocha",
     "lint": "jsinspect src && jsinspect test && eslint src test",
-    "test": "npm run lint && npm run unit-test"
+    "test": "npm run lint && npm run unit-test",
+    "postinstall": "npm install && rimraf .git && git init && babel-node ./scripts/setup.js && npm install --save-dev most"
```

Comment:
I think maybe the only thing left is to remove the leading `npm install &&` here, so that we don't get into an infinite loop!